src/main/java/com/young/shortenerurl/url/presentation/UrlRestController.java

```diff
@@ -0,0 +1,64 @@
+package com.young.shortenerurl.url.presentation;
+
+import com.young.shortenerurl.url.application.UrlService;
+import com.young.shortenerurl.url.application.dto.UrlInfoFindResponse;
+import com.young.shortenerurl.url.presentation.dto.UrlCreateApiRequest;
+import com.young.shortenerurl.url.presentation.dto.UrlCreateApiResponse;
+import com.young.shortenerurl.url.presentation.dto.UrlInfoFindApiResponse;
+import com.young.shortenerurl.url.presentation.mapper.UrlApiMapper;
+
+import jakarta.validation.Valid;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+@RestController
+@RequestMapping("/api/urls")
+public class UrlRestController {
+    private static final String URL_PRE_FIX = "http://localhost:8080/api/urls/";
+
+    private final UrlService urlService;
+    private final UrlApiMapper mapper;
+
+    public UrlRestController(UrlService urlService, UrlApiMapper mapper) {
+        this.urlService = urlService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<UrlCreateApiResponse> createUrl(@RequestBody @Valid UrlCreateApiRequest request) {
```

Comment:
너무 길어지면 내리는 것도 좋습니다~ 
```suggestion
    public ResponseEntity<UrlCreateApiResponse> createUrl(
        @RequestBody @Valid UrlCreateApiRequest request
     ) {
```